examples/baselines/ppo/ppo_rgb.py

```diff
@@ -288,7 +290,7 @@ def close(self):
     device = torch.device("cuda" if torch.cuda.is_available() and args.cuda else "cpu")
 
     # env setup
-    env_kwargs = dict(obs_mode="rgb", control_mode="pd_joint_delta_pos", render_mode="all", sim_backend="gpu")
+    env_kwargs = dict(obs_mode="rgbd", control_mode="pd_joint_delta_pos", render_mode=args.render_mode, sim_backend="gpu")
```

Comment:
why is it changed to rgbd? should be rgb

Reply:
Apologies, it used to be rgbd until #475, and I didn't see that change when resolving the merge conflict. It's fixed now.
The training curve generated above for rgb only didn't include depth regardless, due to the --render_mode="rgb_array" argument, which can be observed in the evaluation video linked under the training curve for it above